pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -449,7 +449,8 @@ func (c *Cluster) determineExtraMonToRemove() string {
 	arbitraryMon := ""
 	for _, mon := range mons {
 		if mon.NodeName == "" {
-			logger.Debugf("mon %q is not scheduled to a specific host", mon.DaemonName)
+			logger.Debugf("mon %q is not scheduled to a specific host, removing portable mon %q", mon.DaemonName, mon.DaemonName)
```

Comment:
Let's actually revert this log change. Since we are in a `for` loop, one of the other mons might be chosen instead to remove, instead of the first portable mon that is found. On line 467 we get the info log about which mon is exactly being removed.

Reply:
okay, reverted